package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "alphagov-wordpress-theme",
+  "version": "1.0.0",
+  "description": "Alpha Gov WordPress Theme",
+  "author": "govwordpress@gov.bc.ca",
+  "license": "Apache-2.0",
+  "keywords": [
+    "WordPress",
+    "Block Theme",
+    "BCGov",
+    "AlphaGov",
+    "Gutenberg"
+  ],
+  "homepage": "https://github.com/bcgov/alphagov-wordpress-theme#readme",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/bcgov/alphagov-wordpress-theme.git"
+  },
+  "main": "index.js",
+  "scripts": {
+    "build": "wp-scripts build src/scripts/public/font.js --output-path=dist",
+    "build:production": "npm ci && npm run fix:js && npm run lint:js && npm run fix:css && npm run lint:css && npm run lint:md:docs && npm run test:unit && npm run build",
+    "lint:css": "wp-scripts lint-style src/styles",
+    "fix:css": "wp-scripts lint-style --fix --config @wordpress/stylelint-config/scss src/styles",
+    "lint:js": "wp-scripts lint-js src/scripts",
+    "fix:js": "wp-scripts lint-js src/scripts --fix",
+    "format": "wp-scripts format src/",
+    "lint:md:docs": "wp-scripts lint-md-docs --config .markdownlint.json --ignore-path .markdownlintignore",
+    "lint:pkg-json": "wp-scripts lint-pkg-json",
+    "packages-update": "wp-scripts packages-update",
+    "start": "wp-scripts start src/scripts/public/font.js --output-path=dist",
+    "test:e2e": "wp-scripts test-e2e --passWithNoTests",
+    "test:unit": "wp-scripts test-unit-js --passWithNoTests",
+    "test": "npm run test:unit",
+  },
+  "engines": {
+    "npm": ">= 10.7.0",
```

Comment:
The ticket mentions an .npmrc file which will prevent npm from running if the npm version isn't high enough. Copying this file should work: https://github.com/bcgov/bcgov-wordpress-block-theme/blob/development/.npmrc

Reply:
good catch. I'll grab that and throw it in

Reply:
on second thought, I noticed it already reacts to the wrong engine: 
![image](https://github.com/bcgov/alphagov-wordpress-theme/assets/122112933/7fae21f4-bcda-4fda-8d39-53e91a7ef450)

maybe it makes more sense not to include the .npmrc (unless theres something else in it we need)

Reply:
I don't feel that strongly about using a .npmrc file but that's what's in the acceptance criteria. It does complain about the unsupported engine without the .npmrc file but it still allows the command to go through. npmrc exits the command right away.